plantcv/plantcv/visualize/__init__.py

```diff
@@ -6,6 +6,8 @@
 from plantcv.plantcv.visualize.auto_threshold_methods import auto_threshold_methods
 from plantcv.plantcv.visualize.overlay_two_imgs import overlay_two_imgs
 from plantcv.plantcv.visualize.colorize_label_img import colorize_label_img
+from . import ecdf
```

Comment:
```suggestion
from plantcv.plantcv.visualize import ecdf
```

I think this is the import we want